Cargo.toml

```diff
@@ -13,7 +13,7 @@ rust-version.workspace = true
 [dependencies]
 arrayvec = { version = "0.7", default-features = false }
 serde = { version = "1.0.217", default-features = false, optional = true }
-parity-scale-codec-derive = { path = "derive", version = "3.6.8", default-features = false, optional = true }
+parity-scale-codec-derive = { path = "derive", version = "=3.7.3", default-features = false, optional = true }
```

Comment:
Makes sense, but can't we do `parity-scale-codec-derive = { path = "derive", default-features = false, optional = true }` ?

Reply:
Oh, ok